opentracing-api/src/main/java/io/opentracing/BuiltinFormats.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright 2016 The OpenTracing Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.opentracing;
+
+import java.nio.ByteBuffer;
+import io.opentracing.propagation.TextMapReader;
+import io.opentracing.propagation.TextMapWriter;
+
+/**
+ * BuiltinFormats encapsulates the inject()/extract() carrier formats that all Tracer implementations must support.
+ *
+ * @see Format
+ * @see io.opentracing.Tracer#inject(SpanContext, Format, Object)
+ * @see io.opentracing.Tracer#extract(Format, Object)
+ */
+public class BuiltinFormats<R, W> implements Format<R, W> {
```

Comment:
@yurishkuro I went with `BuiltinFormats` instead of `Formats` -- seemed to better emphasize that non-builtin extensions were feasible.